support-workers/src/main/scala/com/gu/support/workers/lambdas/PreparePaymentMethodForReuse.scala

```diff
@@ -98,15 +98,15 @@ class PreparePaymentMethodForReuse(servicesProvider: ServiceProvider = ServicePr
       })
     } yield HandlerResult(
       CreateZuoraSubscriptionState(
-        productState,
-        state.requestId,
-        state.user,
-        productType,
-        state.analyticsInfo,
-        None,
-        None,
-        None,
-        None,
+        productSpecificState = productState,
+        requestId = state.requestId,
+        user = state.user,
+        product = productType,
+        analyticsInfo = state.analyticsInfo,
+        firstDeliveryDate = None,
+        promoCode = state.promoCode,
```

Comment:
This is the added line - it just became hard to read without the named parameters.